examples/Textarea.scroll.tsx

```diff
@@ -3,15 +3,16 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { Textarea } from '@itwin/itwinui-react';
+import { LabeledTextarea } from '@itwin/itwinui-react';
 
 export default () => {
   const [value, setValue] = React.useState(
     'If the area is shrunk to a size smaller than the amount of text within, a scrollbar appears to keep navigation possible. This behavior is especially important for textarea without a resizing handle, to avoid making text editing tedious.',
   );
 
   return (
-    <Textarea
+    <LabeledTextarea
+      label='textarea-scroll'
```

Comment:
Usually labels are words/phrases. The id feels a bit odd.

```suggestion
      label='Textarea Scroll'
```